neo/Settings.py

```diff
@@ -19,22 +19,35 @@
 
 from neorpc.Client import RPCClient
 from neorpc.Settings import settings as rpc_settings
-
+import sys
 
 # Create am absolute references to the project root folder. Used for
 # specifying the various filenames.
 dir_current = os.path.dirname(os.path.abspath(__file__))
-DIR_PROJECT_ROOT = os.path.abspath(os.path.join(dir_current, ".."))
+DIR_PROJECT_ROOT = os.getcwd()
```

Comment:
I think this solution is not so great. Imagine I run `np_prompt` from my home directory, and then from any other directory. It will create 2x the Chain directory and start syncing from scratch. I think we should use something like `~/.neopython/Chains` as default directory. [XDG Base Directory](https://wiki.archlinux.org/index.php/XDG_Base_Directory_support) might be helpful for this too.

Reply:
Since `DIR_PROJECT_ROOT` changes it's meaning from project root to current working directory, perhaps we should remove it completely? What's the use case for leaving it in?